ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml

```diff
@@ -1,5 +1,61 @@
 postsubmits:
   openshift/openshift-azure:
+  - agent: kubernetes
+    branches:
+    - master
+    - private-preview-1
+    decorate: true
+    labels:
+      artifacts: images
+    name: branch-ci-openshift-openshift-azure-master-e2e-azure-conformance
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-azure-secret
+        - --template=/usr/local/e2e-azure-conformance
+        - --target=e2e-azure-conformance
+        - --target=[output:stable:sync]
+        command:
+        - ci-operator
+        env:
+        - name: TEST_FOCUS
+          value: Suite:openshift/conformance/parallel
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: master.yaml
+              name: ci-operator-openshift-openshift-azure
+        - name: JOB_NAME_SAFE
+          value: e2e-azure-conformance
```

Comment:
@bbguimaraes does this need to be unique across presubmits and postsubmits?

Reply:
It has to be unique across jobs that will act on the same input, so no.